js/contact.js

```diff
@@ -0,0 +1,24 @@
+
+        var contactForm = document.querySelector('#contactFormInput');
+
+
+        contactForm.addEventListener('submit', (e) => {
+            e.preventDefault();
+
+            db.collection('reviews').add({
+                name: contactForm['nameAuth'].value,
+                email: contactForm['emailAuth'].value,
+                phone: contactForm['phoneAuth'].value,
+                message: contactForm['messageAuth'].value
```

Comment:
This could be created above and passed in the add() function as an argument for clarity but this also works fine.
More things you could improve: 

- the menu is invisible when the content color matches the one in the navbar on scroll. 
- there's a blue line before the resume link in the navbar, you could remove it.